example/spec/parallel/ingest/ingestFromPdrWithChildWorkflowMetaSpec.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+/**
+ * End to end ingest from discovering a PDR
+ *
+ * Kick off discover and queue pdrs which:
+ * Discovers 1 PDR
+ * Queues that PDR
+ * Kicks off the ParsePDR workflow
+ *
+ * Parse PDR workflow:
+ * parses pdr
+ * queues a granule
+ * pdr status check
+ * This will kick off the ingest workflow
+ *
+ * Ingest worklow:
+ * runs sync granule - saves file to file staging location
+ * performs the fake processing step - generates CMR metadata
+ * Moves the file to the final location
+ * Does not post to CMR (that is in a separate test)
+ */
+
+const { randomString } = require('@cumulus/common/test-utils');
+const { deleteS3Object } = require('@cumulus/aws-client/S3');
+const { s3 } = require('@cumulus/aws-client/services');
+const { LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+
+const {
+  addCollections,
+  addProviders,
+  api: apiTestUtils,
+  buildAndExecuteWorkflow,
+  cleanupProviders,
+  cleanupCollections,
+  getExecutionInputObject,
+  waitForStartedExecution,
+} = require('@cumulus/integration-tests');
+
+const {
+  createTestDataPath,
+  createTestSuffix,
+  createTimestampedTestId,
+  deleteFolder,
+  loadConfig,
+  uploadTestDataToBucket,
+  updateAndUploadTestDataToBucket,
+} = require('../../helpers/testUtils');
+
+const lambdaStep = new LambdaStep();
+const workflowName = 'DiscoverAndQueuePdrsChildWorkflowMeta';
+const origPdrFilename = 'MOD09GQ_1granule_v3.PDR';
+
+const s3data = [
+  '@cumulus/test-data/pdrs/MOD09GQ_1granule_v3.PDR',
+];
+
+const unmodifiedS3Data = [
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
+  '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
+];
+
+describe('The DiscoverAndQueuePdrsChildWorkflowMeta workflow', () => {
+  const providersDir = './data/providers/s3/';
+  const collectionsDir = './data/collections/s3_MOD09GQ_006';
+
+  let beforeAllFailed;
+  let config;
+  let pdrFilename;
+  let provider;
+  let testDataFolder;
+  let testSuffix;
+  let workflowExecution;
+  let addedCollection;
+  let executionNamePrefix;
+  let queuePdrsOutput;
```

Comment:
I'm just noticing this about Jasmine (still a newbie). Do we not have the concept of a `context`? Any time we want to refer to and share variables outside of the scope of the `it` block we need to declare it in a higher scope?

Reply:
That is correct.